plugins/serverless/src/tasks/teardown.ts

```diff
@@ -56,10 +35,7 @@ export default class ServerlessTeardown extends Task<{ plugin: typeof Serverless
     }
 
     const child = spawn('serverless', args, {
-      env: {
-        ...process.env,
-        ...vaultEnv
-      }
+      env: process.env
```

Comment:
**nitpick**: this is the [default value of `env`](https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_processspawncommand-args-options) so we don't need to pass any options

Reply:
i think i prefer having it explicit?